swift/aperture/main.swift

```diff
@@ -70,7 +64,7 @@ func usage() {
 let numberOfArgs = CommandLine.arguments.count
 
 if numberOfArgs == 8 {
-  record()
+  try record()
```

Comment:
@sindresorhus I tried using try in global scope outside of a do/catch block like this one more time just to make sure.

Although this compiles it actually throws a fatal if record() throws and error.. You might want to wrap it in a do/catch and handle the error..

Suspect this is a swift compiler bug and should not be allowed. Will file a bug..